src/frontend/utils/video-helpers.js

```diff
@@ -0,0 +1,7 @@
+export const getVideoUrl = id => {
+	return `https://youtu.be/${id}`;
+};
+
+export const getImageUrl = id => {
+	return `https://i3.ytimg.com/vi/${id}/sddefault.jpg`;
```

Comment:
No need to use return here, you can do like this **export const getImageUrl = id => `https://i3.ytimg.com/vi/${id}/sddefault.jpg`;**

Reply:
Yes, will make the changes